### ISSUE TITLE
where to get "__hstc" value

### ISSUE THREAD

**Issue by 4nuG** [khangly/handshook#1]
I was not able to get a "__hstc" value from the list of cookies in storage. Can I run the script with out this value?

**Comment by khangly** [comment]
Yes just try it. The cookie object in the config is by no mean the required structure. I copied it from Handshake, so the cookie structure could have changed since I started the project. Let me know if it works! If it still does not work you probably have to fix the code yourself. It's been a few years so they could have added a rate limiter or Captcha. Feel free to submit a pull request if you find a fix. Thanks!
